common/common-rest/src/main/java/io/servicecomb/common/rest/definition/path/URLPathBuilder.java

```diff
@@ -66,26 +66,26 @@ private void initPathWriterList(String rawPath, Map<String, RestParam> paramMap)
       rawPath = SLASH + rawPath;
     }
 
-    String tmpPath = "";
+    StringBuilder tmpPath = new StringBuilder();
     for (int idx = 0; idx < rawPath.length(); idx++) {
       char currentChar = rawPath.charAt(idx);
       if (currentChar == '{') {
         if (tmpPath.length() != 0) {
-          this.pathParamWriterList.add(new StaticUrlParamWriter(tmpPath));
-          tmpPath = "";
+          this.pathParamWriterList.add(new StaticUrlParamWriter(tmpPath.toString()));
+          tmpPath = tmpPath.delete(0, tmpPath.length());
```

Comment:
tmpPath.setLength(0) is simpler.

Reply:
thanks @wujimin , that's a brilliant idea.